screens/RecipeListScreen.js

```diff
@@ -32,7 +32,7 @@ const RecipeListScreen = (props) => {
           {props.recipes.map((recipe, key) => {
             return (
               <View style={styles.oneCardContainer}>
-                <RecipeCard recipe={recipe} key={key} />
+                <RecipeCard recipe={recipe} key={key} user={props.user}/>
```

Comment:
Got a recommendation here. Instead of passing user from 'recipe list' -> 'recipe card', can't we directly access 'user' in 'recipe card' directly.